### ISSUE TITLE
ArgumentError for routes/analytics.rb

### ISSUE THREAD

**Issue by Therindor** [hashview/hashview#399]
I upgraded to the latest version, using Sequel. Passwords containing non-ascii characters then caused an internal server error when visiting the analytics page. The stacktrace revealed that it was throwing an error when doing the `force_encoding('UTF-8')` on the plaintext in `routes/analytics.rb`.

Some troubleshooting revealed that, adding to `config/database.yml` another line `encoding: "utf8"`, solved the issue, and the password containing non-ascii was then printed to page and no internal server error occurred.

Has anyone experienced a similar issue?

**Comment by i128** [comment]
Yes we have experienced this problem, I was hoping that the force_encoding would fix it. But clearly it didnt. Mind sharing where in the config/database.yml, you made that change?

**Comment by Therindor** [comment]
Certainly. I added this line: `encoding: "utf8"` to the `config/database.yml` , like this:
```
production:
  adapter: mysql
  host: "localhost"
  port: 3306
  user: "root"
  password: "CHANGEME"
  database: "hashview"
  encoding: "utf8"
```

**Comment by i128** [comment]
KK this SHOULD be fixed with the latest version v0.7.4-beta
